lib/dishes.rb

```diff
@@ -3,6 +3,8 @@ class Dishes
   attr_reader :dishes
 
   def initialize
-    @dishes = ["Chicken = £5", "Noodles = £3" , "Steak = £6"]
+    @dishes = ["Chicken = £5",
+               "Noodles = £3",
+               "Steak = £6"]
```

Comment:
I like this

Reply:
Thanks I thought it would look better when you have multiple things in an array to separate them out